ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectIntTest.java

```diff
@@ -117,35 +214,61 @@ public void testSelectProject() throws Exception {
 
 
   @Test
-  public void testSelectStar() throws Exception {
+  public void testSelectProjectAvroJson() throws Exception {
 
-    ksqlContext.sql("CREATE STREAM EASYORDERS AS SELECT * FROM orders;");
+    String resultStream = "PROJECT_STREAM_AVRO";
+    ksqlContext.sql(String.format("CREATE STREAM %s WITH ( value_format = 'JSON') AS SELECT "
+                                  + "ITEMID, "
+                                  + "ORDERUNITS, "
+                                  + "PRICEARRAY FROM %s;", resultStream, avroStreamName));
 
-    Map<String, GenericRow> easyOrdersData = testHarness.consumeData("EASYORDERS", dataProvider.schema(), dataProvider.data().size(), new StringDeserializer(), IntegrationTestHarness.RESULTS_POLL_MAX_TIME_MS);
+    Schema resultSchema = ksqlContext.getMetaStore().getSource(resultStream).getSchema();
 
-    assertThat(easyOrdersData, equalTo(dataProvider.data()));
-  }
+    Map<String, GenericRow> easyOrdersData = testHarness.consumeData(resultStream, resultSchema,
+                                                                     dataProvider.data().size(),
+                                                                     new StringDeserializer(),
+                                                                     IntegrationTestHarness
+                                                                         .RESULTS_POLL_MAX_TIME_MS, DataSource.DataSourceSerDe.JSON);
 
+    GenericRow value = easyOrdersData.values().iterator().next();
+    // skip over first to values (rowKey, rowTime)
+    Assert.assertEquals( "ITEM_1", value.getColumns().get(2).toString());
+  }
 
-  @Test
-  public void testSelectWithFilter() throws Exception {
+  private void testSelectStar(String resultStream, String inputStreamName, DataSource.DataSourceSerDe dataSourceSerDe) throws Exception {
```

Comment:
This should fail the line length rule in checkstyle, i.e., max line length is 100 chars